spec/features/course/assessment/submission/submissions_spec.rb

```diff
@@ -34,6 +35,12 @@
         create(:submission, :graded, assessment: assessment, course: course,
                                      creator: course_staff.user)
       end
+      let!(:staff_attempt_submission) do
```

Comment:
Why use staff?

Reply:
anyone can actually. The reason I use staff is because it's simpler to check if the prefill feature works (no need to logout and login again)

Reply:
Then you are not testing the real use case? Who’s gonna grade staff submission?

Reply:
understand the issue now
actually at first I thought since both of them act in a similar way (but then after some retrospection, it also needs some proof as well). So I modified it accordingly already